optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -432,6 +439,8 @@ def __init__(
         if valid_sets is None:
             raise ValueError("`valid_sets` is required.")
 
+        self.optuna_callbacks = optuna_callbacks
```

Comment:
Do you think the `self.optuna_callbacks` can be a private attribute (i.e., `self._optuna_callbacks`)?
IMO, we do not expect users to update it during optimization, and it would be safe if it is private.

Reply:
Agree 👍 I pushed 945ac04a.